content/docs/user-guide/basic-concepts/stage.md

```diff
@@ -2,7 +2,7 @@
 name: Stage
 match: [stage, stages]
 tooltip: >-
-  A stage represents individual data processes, including their input and
-  resulting output which can be combined to build detailed machine learning
-  pipelines.
+  Stages capture the commands, scripts, or code that a DVC pipeline executes,
```

Comment:
capture -> represents (it was better after all I think)

> that a DVC pipeline executes

it's like defining pipelines though pipelines. Can we do something like "that you would run as part of your project to get the result (e..g. train.py) ..."

Reply:
Done in afe2ca5c7. PTAL